Pipfile

```diff
@@ -49,6 +49,7 @@ schedule = "*"
 sentry-sdk = "*"
 uritemplate = "*"
 uuid = "*"
+shortuuid = "*"
```

Comment:
Do we really need a package to generate a random string?

Reply:
For the records, generating a 10c random string can be achieved like this using stlib:

```py
import random
import string

"".join(random.choice(string.ascii_letters) for i in range(10))
```

Reply:
Haha, we even already have a `random_string` function in `core.lib.text` :)